app/ISC_DHCP_Relay_Agent/doc/README.head.md

```diff
@@ -0,0 +1,7 @@
+Basic monitoring of the [ISC DHCP Relay Agent](https://www.isc.org/downloads/dhcp/).
+
+1. Import the
```

Comment:
This is missing the `## Usage` section header. Also consider moving the usage section to the docs template.

Reply:
Removed and updated with the help of the new default usage generation (#74).